test/powershell/Modules/Microsoft.PowerShell.Utility/Measure-Object.Tests.ps1

```diff
@@ -174,6 +174,18 @@ Describe "Measure-Object" -Tags "CI" {
 
             $actual.Maximum | Should -Be $expected
         }
+
+        It "Should be able to return all the statitics for given values" {
+            $testNumbers = 1,1,2,4,5,6
+            $actual      = $testNumbers | Measure-Object -AllStats
+
+            $actual.Average | Should -Not -BeNullOrEmpty
```

Comment:
These Should statements should be testing for numeric values, not string values. 

Take a look at the test "Measure-Object with ScriptBlock properties should work".  You can use the same range and results for this test.

Reply:
@dantraMSFT Thanks, changed the sane. But the `StandardDeviation` is always a decimal value and cannot be asserted with Int data type, hence excluded.

Reply:
@kvprasoon: I suggest you at least verify it is populated, such as comparing it's ToString() value against the expected string value (2.13697605664328); otherwise, its not completely verified.

Reply:
@dantraMSFT Done..

Reply:
Thanks @kvprasoon, that looks good.